### ISSUE TITLE
Incorrect `rgb` instead of `rgb255`

### ISSUE THREAD

**Issue by jhf** [dawehner/elm-colorbrewer#5]
The new color package uses `rgb` for `0-1` values, and `rgb255` for `0-255` values,
however in for instance https://github.com/dawehner/elm-colorbrewer/blob/master/src/Colorbrewer/Qualitative.elm#L16
there is
```
[ rgb 102 194 165, rgb 252 141 98, rgb 141 160 203 ]
```
while it should have been
```
[ rgb255 102 194 165, rgb255 252 141 98, rgb255 141 160 203 ]
```

Unfortunately there are no compiler error for this mistake, and rather all colors show as white, since all components are bigger than `1`.

**Comment by dawehner** [comment]
Oh, good catch. Do you want to create a PR which fixes the problem?

**Comment by jhf** [comment]
Sure, here you are https://github.com/dawehner/elm-colorbrewer/pull/6

**Comment by dawehner** [comment]
@jhf Sorry for my late response. I've now published version `4.1.1`.
